.github/workflows/e2e-tests-wp-4-7.yml

```diff
@@ -54,6 +54,12 @@ jobs:
       - name: Build
         run: npm run build:test
 
+      # E2E tests use a dedicated container, this is just for Composer.
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.x'
+          tools: composer
+
```

Comment:
I don't think we need this anymore. [Ubuntu 20.04](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#php) (aka `ubuntu-latest`) comes with PHP 7.4.x and Composer 2.0.x preinstalled. We can simply run `php7.4 /usr/bin/composer install --no-interaction --no-progress --no-dev` in the `Composer Install` step and it will work just fine.

Reply:
I didn't realize that the runner came with both PHP 7.4 and 8 (I thought it was only 8 now). I think this step is okay to keep since it runs in literally 1-2s and won't rely as much on the runner installed versions. Maybe this action even uses the same preinstalled binary when run like this?

Let's keep it like this for now to avoid surprises due to changes on what is installed on the runner.